lib/logstash/inputs/sqs.rb

```diff
@@ -83,6 +83,9 @@ class LogStash::Inputs::SQS < LogStash::Inputs::Threadable
   # Name of the SQS Queue name to pull messages from. Note that this is just the name of the queue, not the URL or ARN.
   config :queue, :validate => :string, :required => true
 
+  # Account ID of the AWS account which owns the queue.
+  config :queue_aws_account_id, :validate => :string, :required => false
+
```

Comment:
We should probably use the same naming as the AWS' official documentation and call it ** queue_owner_aws_account_id**, we will need to change the other reference in the code.

Interesting links:
http://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_GetQueueUrl.html
http://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/acp-overview.html